switchsdk/src/androidTest/java/net/gini/switchsdk/uitests/ReviewPictureScreenTest.java

```diff
@@ -97,9 +121,26 @@ public void setUp() throws Exception {
 
     @After
     public void tearDown() {
+        Intents.release();
         SwitchSdk.getSdk().cleanUp();
     }
 
+    private Activity getActivityInstance() {
+        final Activity[] currentActivity = {null};
+
+        getInstrumentation().runOnMainSync(new Runnable() {
+            public void run() {
+                Collection<Activity> resumedActivity =
+                        ActivityLifecycleMonitorRegistry.getInstance().getActivitiesInStage(
+                                Stage.RESUMED);
+                Iterator<Activity> it = resumedActivity.iterator();
+                currentActivity[0] = it.next();
```

Comment:
it's some kind of trick, since it is in a runnable and has to be final. If it is final it can't be assigned therefore a final array is being created.